documentation/planning.md

```diff
@@ -75,38 +75,38 @@ This is the same configuration as shown in the following video (click to open).
 
 [![TesLight Demo](https://img.youtube.com/vi/_N5h1IViB-E/0.jpg)](https://www.youtube.com/watch?v=_N5h1IViB-E)
 
-The following table assumes that the TesLight controller is placed below the centre console for easy access and wiring.
+The following table assumes that the TesLight controller is placed below the center console for easy access and wiring.
 
 | Zone              | LEDs | Type                      | Cable Length | Physical Channel |
 | ----------------- | ---- | ------------------------- | ------------ | ---------------- |
-| Dash              | 72   | LED strip                 | 150cm        | 1                |
-| Centre Console    | 4    | Fiber cable and injectors | 150cm        | 2                |
-| Front Doors       | 130  | Light bar (2 in parallel) | 265cm + 50cm | 3, 4             |
-| Rear Doors        | 92   | Light bar                 | 405cm + 45cm | 5, 6             |
-| Footwell lighting | 4    | LED strip or single LEDs  | 100cm        | 7, 8             |
-
-So for this build we have a total of around 784 LEDs and need ~20m cable with 3 wires.
-Also 5m of fibre cable is recommended for use around the centre console.
+| Dash              | 72   | LED strip                 | 150 cm        | 1                |
+| Center Console    | 4    | Fiber cable and injectors | 150 cm        | 2                |
+| Front Doors       | 130  | Light bar (2 in parallel) | 265 cm + 50 cm | 3, 4             |
+| Rear Doors        | 92   | Light bar                 | 405 cm + 45 cm | 5, 6             |
+| Footwell lighting | 4    | LED strip or single LEDs  | 100 cm        | 7, 8             |
+
+So for this build we have a total of around 784 LEDs and need ~20 m cable with 3 wires.
+Also, 5 m of fibre cable is recommended for use around the centre console.
```

Comment:
I know I wrote this myself. Not looking at it again I feel like 5m is a lot. Wonder why I wrote it back then. Probably because of the lengths that are available on Amazon. Maybe we should check again if we can get away with less waste.

Reply:
lets follow this in #63